themes/digital.gov/layouts/partials/core/get_authors.html

```diff
@@ -35,7 +35,8 @@
           <div class="grid-col-9">
             <div class="details">
               <h5>
-                <a href="{{- (printf "authors/%s/" $uid) | absURL -}} " title="Posts by {{- $author.Params.display_name | default $uid -}} " rel="author">
+                <span class="usa-sr-only">Orginally posted by&nbsp;{{- $author.Params.display_name | default $uid -}}&nbsp;on&nbsp;{{- $.Date.Format "Jan 2, 2006" -}}</span>
+                <a href="{{- (printf "authors/%s/" $uid) | absURL -}} " title="Posts by {{- $author.Params.display_name | default $uid -}} " rel="author" aria-label="Read more articles by&nbsp;{{- $author.Params.display_name | default $uid -}}">
```

Comment:
Can we remove the non-breaking space? This is more for visually getting words on the same line during line breaks.

Reply:
Got it, non-breaking space is removed.